app/controllers/groups/samples_controller.rb

```diff
@@ -22,6 +23,20 @@ def index # rubocop:disable Metrics/AbcSize
       end
     end
 
+    def select
+      authorize! @group, to: :sample_listing?
+      @selected_sample_ids = []
+
+      respond_to do |format|
+        format.turbo_stream do
+          if params[:select].present?
+            @q = authorized_samples.ransack(params[:q])
+            @selected_sample_ids = @q.result.select(:id).pluck(:id)
```

Comment:
**praise:** I guess this would be much faster than what I did on the project samples select.

Reply:
Yeah I was getting a weird model attribute missing when I did it the way you did, but changing to this alleviated the error.